Hydra-Chain/hydragon-node
# 150 integrate new contracts to node

# Description

- update submodules and generate the artifacts and the bindings;
- update the node to use all the new contracts;
- fix tests and compile;
- delete the `--commission` flag for the `register-validator` command;
- rename polygon-edge -> hydra where forgotten;

# Changes include


# Checklist

- [x] I have added sufficient documentation in the code

## Testing


### Manual tests

I have followed the polybft set up in order to check if everything is going smoothly with the new contracts' architecture.
